### DISCUSSION TITLE
How to align radio buttons vertically

### DISCUSSION THREAD

**Discussion by thr3a** [mantinedev/mantine#3908] [Q&A] [answered]
I would like to align radio buttons vertically.
In v5, this could be achieved by setting the orientation of the Radio.Group component to 'vertical'.

I understand that you have read the release notes and realized that orientation support has been removed in v6. However, you are now unsure how to align the radio buttons vertically.

In v6, what is the way to align the radio buttons vertically again?

> [Checkbox.Group](https://mantine.dev/core/checkbox/), [Switch.Group](https://mantine.dev/core/switch/) and [Radio.Group](https://mantine.dev/core/radio/) components no longer include [Group](https://mantine.dev/core/group/) – orientation, offset and spacing props are no longer supported. This change gives you more freedom on how to organize inputs layout.

ref: https://mantine.dev/changelog/6-0-0/

**Comment by rtivital** [comment, marked as answer]
You can use Stack component for that
